File: backend/ideappservice/src/main/java/com/ideapp/ideappservice/model/SubClub.java
package com.ideapp.ideappservice.model;

public class SubClub {
    String name ;
    String clubName;
    String imageUrl;
    String subClubAdmin;
    String description;

    int vote;
    public SubClub(String name, String clubName,String imageUrl) {
        this.clubName = clubName;
        this.name = name;
        this.imageUrl = imageUrl;
    }
    public SubClub(){}

    public int getVote() {
        return vote;
    }
    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setVote(int vote) {
        this.vote = vote;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getClubName() {
        return clubName;
    }

    public String getSubClubAdmin() {
        return subClubAdmin;
    }

    public void setSubClubAdmin(String subClubAdmin) {
        this.subClubAdmin = subClubAdmin;
    }

    public void setClubName(String clubName) {
        this.clubName = clubName;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: frontend/frontend_idea_app/lib/models/SubClub.dart
class SubClub {
  String name, imageUrl, clubName, subClubAdmin, description;
  int vote;

  SubClub(this.name, this.imageUrl, this.clubName, this.subClubAdmin, this.vote,
      this.description);

  SubClub.fromJson(Map<String, dynamic> json)
      : name = json['name'],
        imageUrl = json['imageUrl'],
        clubName = json['clubName'],
        subClubAdmin = json['subClubAdmin'],
        vote = json['vote'],
        description = json['description'];

  Map<String, dynamic> toJson() => {
        "name": "'" + name + "'",
        "imageUrl": "'" + imageUrl + "'",
        "clubName": "'" + clubName + "'",
        "subClubAdmin": "'" + subClubAdmin + "'",
        "vote": vote,
        "description": "'" + description + "'",
      };
}